src/components/Dashboard/FilingHistoryList/HeaderActions.vue

```diff
@@ -110,6 +109,8 @@ export default class HeaderActions extends Mixins(AllowableActionsMixin) {
     conditions[4] = () => (EnumUtilities.isTypeChangeOfRegistration(this.filing) && !this.isFirm)
     conditions[5] = () => (EnumUtilities.isTypeCorrection(this.filing) && !this.isFirm && !this.isBenBcCccUlc)
     conditions[6] = () => (EnumUtilities.isTypeRegistration(this.filing) && !this.isFirm)
+    conditions[7] = () => (EnumUtilities.isTypeSpecialResolution(this.filing) &&
+      EnumUtilities.isStatusCorrected(this.filing))
```

Comment:
Is this to disable the correction of a Special Resolution that has already been corrected?

Is that a special business rule (since our other filings don't have this rules)? We allow multiple corrections to the same filing, since a correction is actually a change to the _current business state_ (not the state as of the filing being corrected). Please confirm.

Reply:
you're correct, corrected SR can be corrected again. 
I've chat with our BA(Jenny), "File a correction" on a correction filing suppose to be disabled. But for other corrections of SP, GP, BEN, UCL, CC they allow correction on a correction filling currently.
so I removed that line, update line 110, let it has same pattern as others.

Reply:
Thanks for checking.

Your code looks fine. I love consistency :)